File: toothless-backend/src/main/java/ru/itis/vhsroni/timetable/service/TimetableService.java
package ru.itis.vhsroni.timetable.service;

import ru.itis.vhsroni.timetable.data.dto.response.TimetableResponse;

import java.time.LocalDate;
import java.util.UUID;

public interface TimetableService {

    TimetableResponse getTimetableByDateRange(UUID dentistId, LocalDate beginDate, LocalDate endDate);
}

File: toothless-frontend/src/services/timetable.service.js
import apiClient from "@/api/axios";

export const TimetableService = {
    async getTimetable(dentistId) {

        try {
            const response = await apiClient.get(`/timetable/${dentistId}`);
            return response.data;
        } catch (err) {
        }

    },
}